### PR TITLE
Onboarding remove identifiers code and refactor create_or_sync_profile screen

### PR DESCRIPTION
**Pull request by Parveshdhull**
### Summary

- PR Removes unused identifiers screen code
- Refactors `create-or-sync-profile` screen and breaks into two different screens (create-profile and log-in)
- Original issue https://github.com/status-im/status-mobile/issues/21650 also had in scope removal of overlay and blur for ios, and was implemented in https://github.com/status-im/status-mobile/commit/dc4ad41342afb2505b08f0c78f050e5ce01e993f. But this change needs to discussed, so removed it from final PR.

### Testing
Please lightly test onboarding flow to make sure cleanup is not breaking it.

status: ready

### CODE REVIEW

File: src/status_im/contexts/onboarding/create_profile/view.cljs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+(ns status-im.contexts.onboarding.create-profile.view
```
Comment (Parveshdhull) [inline src/status_im/contexts/onboarding/create_profile/view.cljs:1]: Refactored the [create-or-sync-profile](https://github.com/status-im/status-mobile/blob/develop/src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs) screen and split it into two separate screens:
('Create Profile' and 'Log In - as per figma)

File: src/status_im/contexts/onboarding/create_profile/view.cljs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  (:require
+    [quo.core :as quo]
+    re-frame.db
```
Comment (flexsurfer) [inline src/status_im/contexts/onboarding/create_profile/view.cljs]: wondering why `re-frame.db` is required

File: src/status_im/contexts/onboarding/log_in/view.cljs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-(ns status-im.contexts.onboarding.create-or-sync-profile.view
+(ns status-im.contexts.onboarding.log-in.view
   (:require
     [quo.core :as quo]
     re-frame.db
```
Comment (flexsurfer) [inline src/status_im/contexts/onboarding/log_in/view.cljs]: `re-frame.db`

File: src/status_im/contexts/onboarding/create_profile/view.cljs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    [status-im.common.events-helper :as events-helper]
+    [status-im.common.resources :as resources]
+    [status-im.config :as config]
+    [status-im.contexts.onboarding.create-profile.style :as style]
+    [status-im.contexts.onboarding.getting-started-doc.view :as getting-started-doc]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- navigate-to-create-profile-password
```
Comment (flexsurfer) [inline src/status_im/contexts/onboarding/create_profile/view.cljs:15]: don we have this in every control ?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/contexts/onboarding/create_profile/view.cljs (hunk 2 above)
Comment (Parveshdhull) [reply src/status_im/contexts/onboarding/create_profile/view.cljs]: Probably some leftover, removed. Thank you

File: src/status_im/contexts/onboarding/create_profile/view.cljs (hunk 4 above)
Comment (Parveshdhull) [reply src/status_im/contexts/onboarding/create_profile/view.cljs:15]: Thank you, @flexsurfer, for reviewing the PR.

Are you asking if we need the create-profile-password in both the create-profile and login flows? Yes, we do.

For the create profile flow, we are directly navigating to the create-profile-password screen here. However, for the login flow, navigation to the password screen occurs after the seed phrase is [validated](https://github.com/status-im/status-mobile/blob/0f036ca9c4fc88f8129cc6ce85bf90733f443f15/src/status_im/contexts/onboarding/events.cljs#L138).

**Review by ilmotta — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
Hey @Parveshdhull !

Tested PR on both platforms.
E2E test looks good.
Thanks for your PR it can be merged.

**Pull request merged**